src/main/scala/no/gdl/bookapi/controller/OPDSController.scala

```diff
@@ -98,12 +109,19 @@ trait OPDSController {
       </feed>
     }
 
-    private[controller] def render(feed: Feed): Elem = {
+    private[controller] def render(feed: Feed, pagingStatus: PagingStatus): Elem = {
       <feed xmlns="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/terms/" xmlns:opds="http://opds-spec.org/2010/catalog" xmlns:lrmi="http://purl.org/dcx/lrmi-terms/">
         <id>{feed.feedDefinition.uuid}</id>
         <title>{feed.title}</title>
         <updated>{feed.updated.atStartOfDay(ZoneId.systemDefault()).format(dtf)}</updated>
         <link href={feed.feedDefinition.url} rel="self"/>
+        {pagingStatus match {
+        case HasMore(currentPage, currentPageSize) =>
+            <link href={s"${feed.feedDefinition.url}?page-size=$currentPageSize&page=${currentPage + 1}"} rel="next"/>
+        case NothingMore =>
```

Comment:
RFC5005 says at least on of "first", "last", "previous", or "next" MUST be present in all paged feed documents.